planning/mission_planner/launch/mission_planner.launch.xml

```diff
@@ -9,6 +9,7 @@
     <param name="arrival_check_distance" value="1.0"/>
     <param name="arrival_check_duration" value="1.0"/>
     <param name="goal_angle_threshold_deg" value="45.0"/>
+    <param name="correct_goal_pose" value="true"/>
```

Comment:
After this PR, could you make another PR to move these parameters to yaml file so that integrators can easily change the parameters for each vehicle.

Reply:
@mehmetdogru Sure, you can assign me.